examples/ubuntu-20.04.yaml

```diff
@@ -0,0 +1,20 @@
+# This template requires Lima v0.7.0 or later.
+images:
+# Try to use release-yyyyMMdd image if available. Note that release-yyyyMMdd will be removed after several months.
+- location: "https://cloud-images.ubuntu.com/releases/20.04/release-20240612/ubuntu-20.04-server-cloudimg-amd64.img"
+  arch: "x86_64"
+  digest: "sha256:7fce49f6c9afd2cdd9c26618b056f8806e500dd4d6eff516c9c6b5fab12e7f79"
+- location: "https://cloud-images.ubuntu.com/releases/20.04/release-20240612/ubuntu-20.04-server-cloudimg-arm64.img"
```

Comment:
It is hard to keep every image up-to-date, so we may need to have an automation script before adding new templates
- https://github.com/lima-vm/lima/issues/1347